error/s2n_errno.c

```diff
@@ -289,7 +292,7 @@ int s2n_error_get_type(int error)
 
 
 /* https://www.gnu.org/software/libc/manual/html_node/Backtraces.html */
-static bool s_s2n_stack_traces_enabled;
+static bool s_s2n_stack_traces_enabled = false;
```

Comment:
Just curious, don't care that you changed it, but was this change actually necessary?
Lord, I hope not 😳 or lots of my code is going to be flaky on Android